libs/@local/hash-backend-utils/src/environment.ts

```diff
@@ -33,3 +33,5 @@ export const waitOnResource = async (
 
   logger?.debug(`${resource} is ready`);
 };
+
+export const realtimeSyncEnabled = !!process.env.ENABLE_REALTIME_SYNC;
```

Comment:
Maybe let's add `ENABLE_REALTIME_SYNC` to the committed `.env` and set it to `false` so it's easier to re-enable this in development down the line. Then we can modify this check to:

```suggestion
export const realtimeSyncEnabled = process.env.ENABLE_REALTIME_SYNC === "true";
```

Reply:
Done in https://github.com/hashintel/hash/pull/3373/commits/8c4346cbc92e67f58ec0fb66fe62179a27327be6